src/content/operations/deployment/subctl/_index.en.md

```diff
@@ -189,6 +189,12 @@ Shows information about the Submariner endpoint connections with other clusters.
 
 Shows information about the Submariner endpoints in the cluster.
 
+### `show brokers`
```

Comment:
I just realized the subcommand is `brokers` but there's only one broker so perhaps we should change it to `broker`.